ext/standard/tests/streams/stream_socket_sendto.phpt

```diff
@@ -0,0 +1,69 @@
+--TEST--
+int stream_socket_sendto ( resource $socket , string $data [, int $flags = 0 [, string $address ]] );
+--CREDITS--
+marcosptf - <marcosptf@yahoo.com.br> - @phpsp - sao paulo - br
+--SKIPIF--
+<?php
+if (getenv("SKIP_ONLINE_TESTS")) { die('skip: online test'); }
+?>
+--FILE--
+<?php
+$serverUri = "tcp://127.0.0.1:31854";
+$serverUrl = "http://php.net";
+$stringSocket = "out of band data.";
+$stringFWrite = "normal data to transmit";
+$sock = stream_socket_server($serverUri, $errno, $errstr);
+
+if (is_resource($sock)) {
+    fwrite($sock, $stringFWrite);
+    var_dump(stream_socket_sendto());
+    var_dump(stream_socket_sendto($sock));
+    var_dump(stream_socket_sendto($sock, $stringSocket));
+    var_dump(stream_socket_sendto($sock, $stringSocket, STREAM_OOB));
+    var_dump(stream_socket_sendto($sock, $stringSocket, STREAM_OOB, $serverUri));
+    var_dump(stream_socket_sendto($sock, $stringSocket, STREAM_OOB, $serverUrl));
+} else {
+    die("Test stream_socket_enable_crypto has failed; Unable to connect: {$errstr} ({$errno})");
+}
+?>
+--CLEAN--
```

Comment:
I forgot to mention, but the entire --CLEAN-- section is not needed, as I explained in another PR, then its run in a separate script after, so the variables do no longer exists, if anything, this should be done in --FILE--, but php already does this once the script ends :)